static/js/vhosts.js

```diff
@@ -27,15 +27,17 @@ function addVhostOptions (formId) {
       }
       const err = document.createElement('span')
       err.id = 'error-msg'
-      err.textContent = "Error fetching data: Please try to refresh the page!"
+      err.textContent = 'Error fetching data: Please try to refresh the page!'
       select.parentElement.insertAdjacentElement('beforebegin',err)
       return
     }
 
+    const selectedVhost = window.sessionStorage.getItem('vhost')
     for (let i = 0; i < vhosts.length; i++) {
       const opt = document.createElement('option')
       opt.text = vhosts[i].name
```

Comment:
```suggestion
```

Reply:
without the text setting, the content of the option tag will be empty

Reply:
I could use the label attribute instead if that is preferred? or is there another way?

Reply:
Ok, docs says that value would be used if no label was set.